recipe/meta.yaml

```diff
@@ -61,13 +61,19 @@ requirements:
     - gsl
     - pixman 0.34.*
     - esmf
+    # This will ensure that ncl works regardles of the gsl downloaded
+    # (with or without openblas) at build time.
+    # We can remove this once we have a better solution.
+    - openblas
```

Comment:
This should cover all bases but we need a better solution to avoid this uncertainty when building against `gsl`.